vmdb/app/models/vm_or_template.rb

```diff
@@ -1160,7 +1160,7 @@ def is_controllable?
 
   def self.refresh_ems(vm_ids)
     vm_ids = [vm_ids] unless vm_ids.kind_of?(Array)
-    vm_ids = vm_ids.collect { |id| [Vm, id] }
+    vm_ids = vm_ids.collect { |id| [base_class, id] }
```

Comment:
So, this is a good surgical fix but I'd prefer we put these two lines into a `refresh_ems_class_and_id_pairs` so we don't have to do "should_receive(:queue_refresh).with....  I'd much rather we test the return value of a method like `refresh_ems_class_and_id_pairs`.